yacl/crypto/primitives/ot/kkrt_ote.cc

```diff
@@ -58,7 +58,7 @@ inline void PrcInit(const std::shared_ptr<link::Context>& ctx,
 }
 
 // Apply Pseudorandom coding on the input
-inline void Prc(AES_KEY* aes_key, uint128_t input, KkrtRow* prc) {
+void Prc(AES_KEY* aes_key, uint128_t input, KkrtRow* prc) {
```

Comment:
Sounds strange, what is the origin build error message?

Reply:
It's an answer mismatch....

Reply:
No error/warning/etc., just got wrong results

Reply:
It's there another symbol (in a different lib) with same name?

Reply:
nope